library/src/main/scala/scala/scalajs/runtime/package.scala

```diff
@@ -97,7 +97,7 @@ package object runtime {
    *
    *  See [[LinkingInfo]] for details.
    */
-  def linkingInfo: LinkingInfo = throw new Error("stub")
+  def linkingInfo: LinkingInfo = LinkingInfo
```

Comment:
This one we will have to remove. MiMa will complain but we can add an exclusion because it was a primitive.